### ISSUE TITLE
Splitting issues and pull requests notifications

### ISSUE THREAD

**Issue by EugeneZelenko** [llvm/llvm-project#64442]
I'm not sure if GitHub allow this, but will be great to split issues and pull requests notifications via e-mail. It'll be not trivial to follow e-mail notifications when code reviews will be migrated to GitHub.

I personally participate only in Clang-tidy related reviews, so fine-grained control for pull request notifications will be very useful.

**Comment by asl** [comment]
@EugeneZelenko can you subscribe to the corresponding label?

**Comment by EugeneZelenko** [comment]
@asl: This will not help me too much because I'm trying to help with issues labeling, so it's mean broad scope in my case.

**Comment by asl** [comment]
Fair enough. New issues got emails sent to llvm-bugs. Will it be working as a "split" in your case?

**Comment by EugeneZelenko** [comment]
@asl: I'll try to play with e-mail filters.
